x-pack/test/api_integration/apis/file_upload/has_import_permission.ts

```diff
@@ -54,6 +54,7 @@ export default ({ getService }: FtrProviderContext) => {
           )
           .auth(IMPORTER_USER_NAME, IMPORT_USER_PASSWORD)
           .set('kbn-xsrf', 'kibana')
+          .set('version', '1')
```

Comment:
Use `ELASTIC_HTTP_VERSION_HEADER` from `import { ELASTIC_HTTP_VERSION_HEADER } from '@kbn/core-http-common';` instead of `version`.

`http.fetch` converts `version` to `ELASTIC_HTTP_VERSION_HEADER` but this does not occur in integration tests.

Reply:
Good spot, thanks. This is correct in previous our versioning PRs, but I missed it here.
Updated in https://github.com/elastic/kibana/pull/158265/commits/957bbca508e1e698281450af26fd5354df066801